test/e2e/virtual/apiexport/authorizer_test.go

```diff
@@ -98,131 +95,198 @@ func TestAPIExportAuthorizers(t *testing.T) {
 	framework.AdmitWorkspaceAccess(ctx, t, kubeClient, tenantPath, []string{"user-3"}, nil, true)
 	framework.AdmitWorkspaceAccess(ctx, t, kubeClient, tenantShadowCRDPath, []string{"user-3"}, nil, true)
 
-	apifixtures.CreateSheriffsSchemaAndExport(ctx, t, serviceProvider1Path, user1KcpClient, "wild.wild.west", "")
-
-	t.Logf("Setting maximal permission policy on APIExport %s|%s", serviceProvider1Path, "wild.wild.west")
-	export, err := user1KcpClient.Cluster(serviceProvider1Path).ApisV1alpha1().APIExports().Get(ctx, "wild.wild.west", metav1.GetOptions{})
-	require.NoError(t, err, "error getting APIExport %s|%s", serviceProvider1Path, export.Name)
+	t.Logf("install sherriffs API resource schema, API export, permissions for user-3 to be able to bind to the export in service provider workspace %q", serviceProvider1Path)
+	require.NoError(t, apply(t, ctx, serviceProvider1Path, framework.UserConfig("user-1", rest.CopyConfig(cfg)),
+		&apisv1alpha1.APIResourceSchema{
```

Comment:
Don't we auto-gen an APIRS from the Go types? Are we using something special here for a reason?

Reply:
there are no go types for these test types (`sherriffs`/`cowboys`) available. these types are just for testing, we don't maintain them.

Reply:
he, i had no idea, however do we want to depend on central fixtures? i was having the impression we want to get rid of it.

Reply:
using those types and clients is fine IMO. "getting rid of the fixtures" is about helper functions built around those types. We don't want them.